### PR TITLE
Legacy DFRN transport layer is removed

### PR DESCRIPTION
**Pull request by annando**
For a long time we don't use the legacy transport layer anymore to transport DFRN messages. For several years we now use the Diaspora transport layer and since some version we mostly use AP.

### CODE REVIEW

File: mod/dfrn_notify.php [hunk 1]
```diff
@@ -53,148 +51,7 @@ function dfrn_notify_post(App $a) {
...
-		Logger::log('rino: decrypted data: ' . $data, Logger::DATA);
-	}
-
-	Logger::log('Importing post from ' . $importer['addr'] . ' to ' . $importer['nickname'] . ' with the RINO ' . $rino_remote . ' encryption.', Logger::DEBUG);
-
-	$ret = DFRN::import($data, $importer, Conversation::PARCEL_LEGACY_DFRN, Conversation::PUSH);
-	System::xmlExit($ret, 'Processed');
-
-	// NOTREACHED
+	throw new HTTPException\BadRequestException();
```
Comment (MrPetovan) [inline mod/dfrn_notify.php:54]: I believe we should add a deprecation message to this error.

File: mod/dfrn_poll.php [hunk 2]
```diff
@@ -20,536 +20,17 @@
...
-				. "\t" . '<status>' . $status . '</status>' . "\r\n"
-				. "\t" . '<dfrn_version>' . DFRN_PROTOCOL_VERSION . '</dfrn_version>' . "\r\n"
-				. "\t" . '<dfrn_id>' . $encrypted_id . '</dfrn_id>' . "\r\n"
-				. "\t" . '<challenge>' . $challenge . '</challenge>' . "\r\n"
-				. '</dfrn_poll>' . "\r\n";
-			exit();
-			// NOTREACHED
-		}
-	}
+	throw new HTTPException\BadRequestException();
```
Comment (MrPetovan) [inline mod/dfrn_poll.php:35]: I believe we should add a deprecation message to this error.

**Review by MrPetovan — CHANGES_REQUESTED** [review]

File: mod/dfrn_notify.php (hunk 1 above)
Comment (annando) [reply mod/dfrn_notify.php:54]: Who would read this? And AFAIK there is no "deprecated" HTTP error code, isn't it?

Comment (MrPetovan) [reply mod/dfrn_notify.php:54]: I meant adding a meaningful message but I guess it isn't necessary.

**Review by MrPetovan — APPROVED** [review]

**Pull request merged**